.evergreen/install-node.sh

```diff
@@ -32,9 +32,27 @@ else
   [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"
 
   set +x # nvm is very verbose
-  echo nvm install --no-progress $NODE_JS_VERSION && nvm alias default $NODE_JS_VERSION
-  nvm install --no-progress $NODE_JS_VERSION
-  nvm alias default $NODE_JS_VERSION
+
+  # A few distros where pre-built node20 does not work out of the box and hence
+  # needs to be built from source
+  if [[ "${DISTRO_ID}" =~ ^(amazon|debian9|rhel7|suse|ubuntu18) ]];
+  then
+    # Building from source requires up to date toolchain in path
+    ORIGINAL_PATH="${PATH}"
+    export PATH="/opt/mongodbtoolchain/v4/bin:${ORIGINAL_PATH}"
+    export CC=gcc
+    export CXX=c++
```

Comment:
These should be either

```suggestion
    export CC=gcc
    export CXX=g++
```

or 

```suggestion
    export CC=cc
    export CXX=c++
```

or

```suggestion
    export CC=clang
    export CXX=clang++
```

but not a (potential) mixture of compilers

Reply:
That sounds right, I actually ended up following what was in [setup-env](https://github.com/mongodb-js/mongosh/blob/main/.evergreen/setup-env.sh#L21-L22).

If that is a mistake, I should probably update it there as well?

Reply:
Yes, I think that’s a typo :)